lib/calculated.js

```diff
@@ -4,64 +4,108 @@ var debug = require('debug')('loopback-ds-calculated-mixin');
 var _ = require('lodash');
 var Promise = require('bluebird');
 
+function getPropertyConfig(property) {
+  if (typeof property === 'string') {
+    return {
+      callback: property,
+    };
+  }
+  return property;
+}
+
 module.exports = function(Model, options) {
 
   // Trigger a warning and remove the property from the watchlist when one of
```

Comment:
This part of the comment seems to be misleading `remove the property from the watchlist `  as it does not seem to remove anything from the watchlist.  This is not introduced by you PR, was like this already.